test/cpp/common/alarm_cpp_test.cc

```diff
@@ -40,6 +42,23 @@ TEST(AlarmTest, RegularExpiry) {
   EXPECT_EQ(junk, output_tag);
 }
 
+TEST(AlarmTest, Move) {
+  CompletionQueue cq;
+  void* junk = reinterpret_cast<void*>(1618033);
```

Comment:
Where is the move here?

Why not:
Alarm first(...);
Alarm second(move(first));
cq.Next(...);
EXPECT(got junk);